mreg_cli/policy.py

```diff
@@ -402,7 +402,7 @@ def _print(key, value, padding=20):
     callback=list_atoms,
     flags=[
         Flag('name',
-             description='Atom name filter (regexp)',
+             description='Atom name, or part of name. You can use * as a wildcard.',
```

Comment:
The text here is a bit unclear. Do I need to use a wildcard to search for part of name? If I need to use a wildcard, the comment should be more like the one on line 310, if not, what purpose does the wildcard serve?

Reply:
That works. Although, as the backend is a regex, don't we realistically also support stuff like 'atom_[abc]*'?

Reply:
I honestly find it odd that we rewrite regex at all. From a users perspective I would expect there to be three ways to search for something:

- Exact (equals foo=value, which is equivalent to foo__exact=value)
- Substring (equals foo__contains=value)
- Regex (equals foo_regex=value)

I feel like we are trying to create a query language of our own without a clear syntax to work from, and I wonder what the username actually expects.

Reply:
Alright so currently we already have the commands `host find`, `network find`, `permission network_list` and `policy list_atoms/list_roles` that support a form of wildcards where a star matches one or more characters. None of the commands support regex, even though the mreg API does.

Feedback from users is that it's unclear where they can use wildcards, and whether it's necessary if supplying only part of a name ( = if the command expexts a substring or exact term).

The aim of this PR was to improve the help texts to clarify that, and make it more consistent across different commands. For the time being, let's see what we can do without rewriting too much.
Many commands expect a substring now, but you can also write things like "foo*bar". In this PR I've tried to make most commands take a substring where it makes sense (instead of the whole term), and change the help texts accordingly.

Reply:
That makes a ton of sense. Thanks!